examples/salt_transport/src/example_three_boxes_ZSF.py

```diff
@@ -0,0 +1,386 @@
+import copy
+import logging
+
+import numpy as np
+import pandas as pd
+import pyzsf
+from rtctools.simulation.csv_mixin import CSVMixin
+from rtctools.simulation.simulation_problem import SimulationProblem
+from rtctools.util import run_simulation_problem
+from rtctools_interface.simulation.plot_mixin import PlotMixin
+
+from rtctools_simulation.salt_simulation_mixin import SaltSimulationMixin
+
+logger = logging.getLogger("rtctools")
+
+
+def zsf_routines(
+    self, routine, t_level, t_open_lake, t_flushing, t_open_sea, salinity_lake, parameters
+):
+    level_to_fresh_side = 1
+    door_open_on_fresh_side = 2
+    leveling_to_salt_side = 3
+    door_open_on_salt_side = 4
+    flushing_2 = -2
+    flushing_4 = -4
+    if routine == level_to_fresh_side:
```

Comment:
I think it would be very useful to add some docstrings/comments here at least explaining how to call it and what the options are. I wouldn't know what the possible options are for `routine` for example. It's also not very clear to me what this function does, exactly, but that could also be my lack of familiarity with RTC-tools.